mlir/lib/Target/LLVMIR/ModuleImport.cpp

```diff
@@ -2544,6 +2557,12 @@ LogicalResult ModuleImport::processInstruction(llvm::Instruction *inst) {
   if (auto *intrinsic = dyn_cast<llvm::IntrinsicInst>(inst))
     return convertIntrinsic(intrinsic);
 
+  // Capture instruction with attached debug markers for later processing.
+  if (inst->DebugMarker) {
+    for (llvm::DbgRecord &dr : inst->DebugMarker->getDbgRecordRange())
+      addDebugRecord(&dr);
+  }
```

Comment:
```suggestion
  if (inst->DebugMarker) 
    for (llvm::DbgRecord &debugRecord : inst->DebugMarker->getDbgRecordRange())
      addDebugRecord(&debugRecord);
```
nit: the braces can be dropped here.